tools/amd_build/build_amd.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import os
+import sys
+import argparse
+
+sys.path.append(os.path.realpath(os.path.join(
+    os.path.dirname(__file__),
+    os.path.pardir,
+    os.path.pardir,
+    'third_party')))
+
+from hipify import hipify_python
+
+parser = argparse.ArgumentParser(description='Top-level script for HIPifying, filling in most common parameters')
+parser.add_argument(
+    '--project-directory',
+    type=str,
+    help="The root of the project. (default: %(default)s)",
+    required=True)
+
+parser.add_argument(
+    '--output-directory',
+    type=str,
+    help="The Directory to Store the Hipified Project",
+    required=True)
+
+args = parser.parse_args()
+
+includes = [
+    '*'
+]
+
+ignores = [
+]
+
+# capturing the return value which is a dict[filename]:HipifyResult
+HipifyFinalResult = hipify_python.hipify(
+    project_directory=args.project_directory,
+    output_directory=args.output_directory,
+    includes=includes,
+    ignores=ignores,
+    is_pytorch_extension=True)
```

Comment:
What does the `is_pytorch_extension` flag do? TensorPipe is not a PyTorch extension.

Reply:
Yes, tensorpipe is not a extension, but this parameter is used internal in hipify.
I will let @jithunnair-amd to comment for info here.

Reply:
That nomenclature made sense until now :) Essentially, we needed a way to preserve the "legacy" way of hipifying for PyTorch source code, but use the "new and improved" way for other code which used hipify, which were basically PyTorch extensions so far. We'll assess the best way to generalize this, but this flag is needed for now until that happens.

Reply:
Ok, got it, just wanted to check. It would be nice if we could add a comment to explain this, so I won't wonder about it again the next time I read this code. (Though if this code is moved to the hipify repo it won't matter anyways)